src/pages/SignUp/components/ConfirmPasswordInput.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
```

Comment:
prispazinsiu viena dalyka, as visiskai nematau prasmes tureti du atskirus komponentus `PasswordInput` ir `ConfirmPasswordInput`, jie ziauriai identiski. Reikia tureti tiesiog `PasswordInput`, kuris bus completly vizualus, ty jis neturi tureti jokios logikos, tu tiesiog paduoti per propsus dalykus - label, input, onChange, error and so on

Reply:
Awkward ir unclean validacijos funcija pasidaro. Duplicate kodas nevisada yra blogas.